content/en/tracing/trace_collection/library_config/_index.md

```diff
@@ -3,14 +3,113 @@ title: Configure the Datadog Tracing Library
 type: multi-code-lang
 ---
 
-For information about configuration options for tracing libraries, choose your language:
+This page describes configuration options that are supported across all SDKs. To view these common configuration options, see [Common configuration options](#common-configuration-options).
+
+For configuration options specific to your programming language, choose your SDK from the options below:
 
 {{< partial name="apm/apm-compatibility.html" >}}
 
 <br>
 
 To instrument an application written in a language that does not yet have official library support, see the list of [community tracing libraries][1].
 
+## Common configuration options
+The following configuration options are supported across all Datadog SDKs:
```

Comment:
Can we change the wording of "supported" to something else? Here it says "supported across all Datadog SDKs" then later some items have "Not Supported In". I understand that the intent is 1) these are implemented across multiple Datadog SDKs and 2) where implemented, the behavior is consistent, so we need to phrase that in a concise way 😆

Reply:
We could also let the docs team chime in on this, maybe they will have a good way of phrasing this

Reply:
How about something like this?
> The following configuration options behave consistently across multiple Datadog SDKs: